boot/bootutil/src/bootutil_public.c

```diff
@@ -57,7 +57,7 @@ BOOT_LOG_MODULE_DECLARE(mcuboot);
 BOOT_LOG_MODULE_REGISTER(mcuboot_util);
 #endif
 
-#if BOOT_MAX_ALIGN == 8
+#if (BOOT_MAX_ALIGN == 8) || (BOOT_MAX_ALIGN == 16)
```

Comment:
Please check the design doc's trailer section (https://github.com/mcu-tools/mcuboot/blob/main/docs/design.md#image-trailer) at point 7.

IMHO this modification is not needed.

Reply:
I can confirm that mcuboot runs just fine on STM32U5 using the max align options in both the configuration and the imagetool. 
Only the naming is misleading, since actually it is a minimal alignment requirement, not a maximum.